celery_tasks.py

```diff
@@ -71,6 +72,16 @@ def create_pull_request_for_issue(*, token, owner, repo_name, issue, comments=No
 
         files_list = extract_files_list_from_issue(issue['body'])
 
+        # Check for conventions file
```

Comment:
@Aiderbot this is reading the conventions file path outside the GIt repo of the current request. Instead, we want to check that a conventions file exists within the Git repo after we clone the repo. That way, we can specify project-specific conventions. So we shouldn't be writing a new file to the path, we should just be checking if the file exists, and if it does, pass the path through to Aider. Don't worry about reading the contents.